source/entities/signUp/atoms/CheckBoxElements.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import { StyleSheet, Text, View} from 'react-native';
+import { CheckBox } from 'react-native-elements';
+
+export default function CheckBoxElements() {
+    const [privacySelected, privacySelection] = useState(false);
+    const [suscribeSelected, suscribeSelection] = useState(false);
+    return (
+        <View>
+            <View style={styles.checkboxContainer}>
+                <CheckBox
+                    checked={privacySelected}
+                    onPress={() => privacySelection(!privacySelected)}
+                />
+                <Text style={styles.checkBoxText}>I agree to the <Text style = {{textDecorationLine: 'underline'}}>Terms</Text> and <Text style = {{textDecorationLine: 'underline'}}>Privacy Policy.</Text></Text>
+                <Text style={styles.alert}>*</Text>
+            </View>
+
+            <View style={styles.checkboxContainer}>
+                <CheckBox
+                checked={suscribeSelected}
+                onPress={() => suscribeSelection(!suscribeSelected)}
+                />
```

Comment:
You didn't add a tabulation here, but you did above

Reply:
Thanks for the observation, I will make the change.